frontend/src/metabase/parameters/components/ParameterLinkedFilters/ParameterLinkedFilters.tsx

```diff
@@ -111,12 +116,12 @@ const ParameterLinkedFilters = ({
 };
 
 interface LinkedParameterProps {
-  parameter: UiParameter;
```

Comment:
`UiParameter` is dashboard-specific. Cards have `ParameterWithTarget`. Both extend `Parameter`.